charts/tfy-karpenter-config/templates/default/karpenter-default-nodepool.yaml

```diff
@@ -0,0 +1,88 @@
+{{- if .Values.karpenter.defaultNodePool.enabled }}
```

Comment:
Either we should remove this condition, or if we want to keep it then we should add it for `default` ec2 node class also